src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateDirectionsQuestionnaireCallbackHandler.java

```diff
@@ -46,17 +49,68 @@ private CallbackResponse prepareDirectionsQuestionnaire(CallbackParams callbackP
         CaseData caseData = callbackParams.getCaseData();
         CaseData.CaseDataBuilder caseDataBuilder = caseData.toBuilder();
 
-        CaseDocument directionsQuestionnaire = directionsQuestionnaireGenerator.generate(
-            caseData,
-            callbackParams.getParams().get(BEARER_TOKEN).toString()
-        );
+        if (respondent2HasSameLegalRep(caseData)) {
```

Comment:
Ideally this code should be wrapped in a `isMultipartyEnabled` Launch Darkly feature toggle..